src/tsVariableTypes/task2.ts

```diff
@@ -0,0 +1,8 @@
+const variableArray: string[] = ["Hello", "nice", "typeScript"];
+const numberArray: number[] = [];
+
+for (let i = 0; i < variableArray.length; i++) {
```

Comment:
Siūlau šitą patobulinti - pabandykit padaryti be `for` ciklo, o panaudojant array metodus. Ir visada daug geriau naudoti array metodus, o ne for panašiose situacijose.